src/match/match-item.html

```diff
@@ -96,9 +113,40 @@
       static get is() { return 'match-item'; }
       static get properties() {
         return {
-          match: Object
+          match: Object, 
+          player1Color: {
+            type: String,
+            computed: '_applyColor(player1Diff)'
+          },
+          player1Diff: {
+            type: Number,
+            computed: '_calcPlayer1Diff(match)'
+          },
+          player2Color: {
+            type: String,
+            computed: '_applyColor(player2Diff)'
+          },
+          player2Diff: {
+            type: Number,
+            computed: '_calcPlayer2Diff(match)'
+          }
         };
       }
+
+      _calcPlayer1Diff(match) {
```

Comment:
this and `_calcPlayer2Diff` can be a single method - the way u did it with `_applyColor`, just pass different arguments ie.  `f(match.ratings.player2Prev,match.ratings.player2New)`